content/docs/reference/pkg/nodejs/pulumi/pulumi/asset/_index.md

```diff
@@ -32,7 +32,7 @@ git_sha: "03e0005fe0ce4b6bbe04c460a9bbc62869558214"
 
 <h2 id="apis">APIs</h2>
 <h3 class="pdoc-module-header" id="Archive" data-link-title="Archive">
-    <a href="https://github.com/pulumi/pulumi/blob/{{< param git_sha >}}/sdk/nodejs/asset/archive.ts#L21">
+    <a href="https://github.com/pulumi/pulumi/blob/{{< param git_sha >}}/sdk/nodejs/code/go/src/github.com/pulumi/pulumi/sdk/nodejs/asset/archive.ts#L21">
```

Comment:
These don’t look right - do these links work?

Reply:
Fixed - not sure why this happened before!